rlclientlib/ranking_event.h

```diff
@@ -46,13 +50,15 @@ namespace reinforcement_learning {
     const std::vector<float>& get_probabilities() const;
     const std::string& get_model_id() const;
     bool get_defered_action() const;
-  
+    const std::string& get_event_id() const {return get_seed_id();}
```

Comment:
It is weird that get_event_id is re-using get_seed_id - they accidentally share implementation right now, but there is no real dependency between them.

Reply:
Spoke offline - the overload of this value is something to keep in mind as we continue to work on these objects